domains/snnttechv.is-local.org.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "snnttechv.is-local.org",
+    "subdomain": "*snnttechv",
```

Comment:
If you are wanting to use a `*` wildcard you will need to make it `"*.snnttechv"` for it to work. But we will also need a reason to why you need it.

Reply:
ok i fix this, SiR

Reply:
*.snnttechv  is right? Sir

Reply:
If you are wanting to add `*` a wildcard to your domain then yes.

Reply:
It's my first time; I wanted to use this. SIR.